projects/interfaces/buffer/buffer_test.go

```diff
@@ -0,0 +1,73 @@
+package OurBuffer
+
+import (
+	"io"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// If you make a buffer named b containing some bytes, calling b.Bytes() returns the same bytes you created it with.
+func TestBufferBytesReturnsInitialBytes(t *testing.T) {
+	inputBytes := []byte("Hello, World!")
+	b := NewBuffer(inputBytes)
+
+	bytes := b.Bytes()
+
+	excepted := string(inputBytes)
+	actual := string(bytes)
+
+	require.Equal(t, excepted, actual)
+}
+
+// If you write some extra bytes to that buffer using b.Write(), a call to b.Bytes() returns both the initial bytes and the extra bytes.
+func TestBufferWriteExtraBytes(t *testing.T) {
+	inputBytes := []byte("Hello ")
+	b := NewBuffer(inputBytes)
+	n, err := b.Write([]byte("World!"))
+	require.NoError(t, err)
+
+	bytes := b.Bytes()
+	expected := string((bytes))
+	actual := string(inputBytes) + "World!"
```

Comment:
Expected and actual here are the wrong way around :)

Also, I would probably hard-code "Hello World!" rather than using `string(inputBytes) + "World!")` - because this is a test (and we don't test tests), I think reading a complete string is more easy to manually verify when reading than having to look back to a variable assignment, do a conversion in your head, and do an append in your head. e.g. it could be really easy to not notice a missing or extra space or something.